templates/.circleci/config.yml.tpl

```diff
@@ -72,7 +77,50 @@ workflows:
 {{- end }}
   ### End workflows inserted by other modules
 
+  rebuild-cache:
+    triggers:
+      - schedule:
+          # Every day at 00:00 UTC.
+          cron: "0 0 * * *"
+          filters:
+            branches:
+              only:
+                - main
+    jobs:
+      - shared/save_cache:
+          context: *contexts
+          app_name: devbase
+          ### Start parameters from test
```

Comment:
Maybe add a note where the cannoical source of this is? Or move the test params to a higher level key like we did with contexts?

Reply:
I actually never knew that you could re-use blocks like that in yaml. That looks much cleaner, thanks!